schemas/fingerprint-server-api.yaml

```diff
@@ -1768,3 +1834,67 @@ components:
       required:
         - code
         - message
+    EventUpdateRequest:
+      type: object
+      properties:
+        linkedId:
+          type: string
+          description: LinkedID value to assign to the existing event
+        tags:
+          type: object
+          description: Full tags value to be set to the existing event. Replaces any existing tags payload completely.
+        suspect:
+          type: boolean
+          description: Suspect flag indicating observed suspicious or fraudulent event
+    ErrorUpdateEvent400Response:
```

Comment:
@petrpalata 
It seems when I supply a wrong JSON with multiple keys, I get an invalid JSON will multiple errors in the response: 

<img width="927" alt="image" src="https://github.com/fingerprintjs/fingerprint-pro-server-api-openapi/assets/11249993/59332631-f461-4b8b-8577-8e0492aa24d4">

Reply:
Also passing unexpected but valid JSON produces "invalid json" error message 

<img width="914" alt="image" src="https://github.com/fingerprintjs/fingerprint-pro-server-api-openapi/assets/11249993/3c45a6e5-0cac-4df4-aec4-b0cc83473df8">

Reply:
changed this to something more generic "request body is not valid", as it doesn't conform.

Reply:
Nice, those issues have been resolved, but the message still says `must provide one of: linkedId, suspect, tags to be updated` even though the property is `tag` (not `tags`)

Reply:
Let us fix that on the backend next week, I wouldn't be afraid to merge this though because it's a very minor thing.